README.md

```diff
@@ -1,19 +1,19 @@
-Dune - A composable build system
+Dune - A Composable Build System
 ================================
 
 Dune is a build system designed for OCaml/Reason projects only. It
 focuses on providing the user with a consistent experience and takes
-care of most of the low-level details of OCaml compilation. All you
-have to do is provide a description of your project and dune will
+care of most low-level details of an OCaml compilation. It's merely necessary 
```

Comment:
I'm not sure about the `an` here. It was meant to talk about the process of compiling OCaml programs in general.

Reply:
Perhaps "...care of most low-level details of OCaml compilations." ...make compilation plural? If that's still not accurate, it's fine to leave it as it is.

Reply:
Yes, making it plural seems more natural to me.